File: Exercises(Haskell)/exercise1.hs

estacionamento :: String -> Int
estacionamento n | (n == "moto") = 5
		 | (n == "carro") = 10	
		 | (n == "largo") = 30

main = do
	 name <- getLine
	 putStrLn(show (estacionamento name))
	

File: Exercises(Prolog)/exercise1.pl
estacionamento(moto,5).
estacionamento(carro,10).
estacionamento(largo,30).

:- initialization main.

main :-
	read_line_to_codes(user_input, A2),
	string_to_atom(A2,A1),
	estacionamento(A1, Y),
	write(Y).

